tests/FakeItEasy.Tests/Creation/ProxyGeneratorSelectorTests.cs

```diff
@@ -92,16 +93,15 @@ public void Should_delegate_calls_to_default_generator_asking_about_non_delegate
             var getHashCode = fake.GetType().GetMethod("GetHashCode");
 
             string reason = null;
-            A.CallTo(() => this.defaultProxyGenerator.MethodCanBeInterceptedOnInstance(getHashCode, fake, out reason))
-                .Returns(true).AssignsOutAndRefParameters("reason");
+            A.CallTo(() => this.defaultProxyGenerator.MethodCanBeInterceptedOnInstance(getHashCode, fake, out reason)).Returns(true).AssignsOutAndRefParameters("reason");

```

Comment:
I've no problem with the preferred line length becoming 120. I'm not that fussy.
(And I should probably also admit that for me, the whole statement is visible in the GitHub UI. Landscape mode, 1920px, 12px Consolas. I do use [the github expandizr](https://github.com/thecodejunkie/github.expandinizr), though. Maybe that's the difference.)